src/reynir_correct/pattern.py

```diff
@@ -590,6 +610,18 @@ def maybe_place(tree: SimpleTree) -> bool:
             )
         )
 
+        # Check use of "vera að" instead of a simple verb
+        cls.ctx_vera_að = {"verb": "@'vera'",}
```

Comment:
A macro is not needed if only one verb will ever match the rule, in this case 'vera'.

Reply:
Ah, good to know! Meant to include a "nhm": "@'að'"  but ultimately didn't need it.